src/main/scripts/org/reaktivity/specification/sse/challenge/respond.to.challenge/request.rpt

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright 2016-2019 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+connect await ROUTED_SERVER
+        "nukleus://streams/sse#0"
+        option nukleus:capabilities ${core:capabilities("CHALLENGE")}
+        option nukleus:window 8192
+        option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${http:beginEx()
+                              .typeId(nukleus:id("http"))
+                              .header(":scheme", "http")
+                              .header(":method", "GET")
+                              .header(":path", "/events/a8b7c6d5")
+                              .header(":authority", "localhost:8080")
+                              .header("accept", "text/event-stream")
+                              .build()}
+
+connected
+
+read nukleus:begin.ext ${http:beginEx()
+                             .typeId(nukleus:id("http"))
+                             .header(":status", "200")
+                             .header("content-type", "text/event-stream")
+                             .build()}
+
+read nukleus:challenge
+write "type:challenge\n\n"
```

Comment:
Please add `${http:challengeEx().header(...).build()` to `read nukleus:challenge` and corresponding JSON to `write "type:challenge\n\n\data:JSON\n\n"`.